### PR TITLE
Updates new CircuitVerse logo.

### LINKED ISSUE #1387: SVG Logo update
**Describe the bug**
The new SVG Logo needs to be updated in the header and footer sections of thew website.

**To Reproduce**
1. Go to 'http://circuitverse.org'
2. Scroll up to see the header.
3. Scroll down to see the footer.

**Expected behavior**
The logo displayed is in SVG format without advertent white marks.

**Desktop (please complete the following information):**
 - OS: Ubuntu 18.04
 - Browser: Chrome

**Are you working on this issue? (Yes/No)**
Yes.

### PR DESCRIPTION
**Pull request by ShubhankarKG**
Fixes #1387 , #1382

#### Describe the changes you have made in this PR -
The new SVG logo without the white marks has been updated in two files :-

1. _header.html.erb
2. _footer.html.erb

### Screenshots of the changes (If any) -
Screenshot from 2020-04-10 09-00-39

Screenshot from 2020-04-10 09-00-43

### CODE REVIEW

**Comment by ShubhankarKG** [comment]
The teachers.html.erb got affected. I'll revert the same.

**Comment by ShubhankarKG** [comment]
@nitin10s please review.

**Comment by aayush-05** [comment]
Can you link the #1382 issue as well, it is regarding the same bug as well ?

**Comment by ShubhankarKG** [comment]
Done.

**Comment by ShubhankarKG** [comment]
Is there anything more that should be done here ? @nitin10s @aayush-05 ?

**Comment by aayush-05** [comment]
LGTM, @tachyons can be merged

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<img src="/public/img/cvlogo.svg" alt="The CircuitVerse logo" width="736"/> </br></br>
+<img src="/app/assets/images/cvlogo.svg" alt="The CircuitVerse logo" width="736"/> </br></br>
```
Comment (satu0king) [inline README.md:1]: /app/assets cannot be hardcoded

Comment (tachyons) [reply README.md:1]: This is readme, so not an issue. But hardcoding app/assets inside the app won't work

Comment (ShubhankarKG) [reply README.md:1]: exactly, since README is outside the app, I needed to hard code that.

**Comment by ShubhankarKG** [comment]
Does it really need one ? As @tachyons mentioned, this was needed.

**Pull request merged**